Remarkable-Donkeys/bangazon
# Kb products

# Steps To Test #
1. Check Product Model - does it look right
2. Check Product Controller. It should allow for: GET (single and list), POST, PUT, DELETE 

[Hyperlink to ticket]https://github.com/Remarkable-Donkeys/bangazon/issues/2

### Description of what should happen when steps are followed ###

This is an initial pull request to provide blank models to teammates. Please take a look to see if anything looks wrong.

**At least one person needs to approve pull request before merged to master**
